improver_tests/synthetic_data/test_set_up_test_cubes.py

```diff
@@ -690,6 +693,251 @@ def test_equalarea_domain_corner(self):
         )
 
 
+class Test_set_up_spot_variable_cube(IrisTest):
+    """Test the set_up_spot_variable_cube base function. Much of this
+    functionality overlaps with the gridded case, so these tests primarily
+    cover the spot specific elements."""
+
+    def setUp(self):
+        """Set up simple temperature data array"""
+        self.data = np.linspace(275.0, 284.0, 4).astype(np.float32)
+        self.data_2d = np.array([self.data, self.data, self.data])
+        self.site_crds = ["latitude", "longitude", "altitude", "wmo_id"]
+
+    def test_defaults(self):
+        """Test default arguments produce cube with expected dimensions
+        and metadata"""
+        result = set_up_spot_variable_cube(self.data)
+
+        # check type, data and attributes
+        self.assertIsInstance(result, iris.cube.Cube)
+        self.assertEqual(result.standard_name, "air_temperature")
+        self.assertEqual(result.name(), "air_temperature")
+        self.assertEqual(result.units, "K")
+        self.assertArrayAlmostEqual(result.data, self.data)
+        self.assertEqual(result.attributes, {})
+
+        # check dimension coordinates
```

Comment:
```suggestion
        # check auxiliary coordinates
```
I think that this is checking that the auxiliary coordinates have been assigned to the correct dimension, rather than checking something about the dimension coordinate itself. I might have misunderstood something and it isn't important at all, though.